src/Curve/mappings/GaugeController.ts

```diff
@@ -49,7 +49,8 @@ export function handleVoteForGauge(event: VoteForGaugeEvent): void {
     event.block.timestamp,
     event.address, // controller
     event.params.gauge_addr, // gauge
-    event.params.weight, // gaugeWeight
+    // event.params.weight is the user_weight for this vote -> ignore it
```

Comment:
```suggestion
```
I think the note of ignoring `weight` (gaugeWeight) can be in the commit message rather than an inline comment.